CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributuing
+
+Contributions to this project are welcome, whether minor updates to documentation or entirely new features.
+This document outlines the technical and procedural requirements for your work to be included.
+All contributions, in any form, must also confirm with the [code of conduct](CODE_OF_CONDUCT.md).
+
+> :warning: This document is a living document, and will adapt based on the project's needs.
+> If some part of this document creates a hindrance, raise an issue explaining the problem.
+
+## Scope: What Lives in this Repository?
+
+MewBot's core design includes the tenant of modularity.
```

Comment:
This implies we have a design philosophy with tenants. We sort of do in the design docs, but it might be cool to codify that some more